l10n-pocket/en/pocket/jobs.ftl

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+## Contact info URL: https://dev.tekcopteg.com/jobs/
```

Comment:
**Nit:** 
```suggestion
## Jobs URL: https://dev.tekcopteg.com/jobs/
```

Also, since this is a file-level comment, it should be 3 hashes:
```suggestion
### Jobs URL: https://dev.tekcopteg.com/jobs/
```